terraform/overrides/nodes_use_ephemeral_storage_override.tf

```diff
@@ -0,0 +1,30 @@
+resource "openstack_compute_instance_v2" "node_server" {
+  block_device {
+    boot_index            = 0
+    delete_on_termination = true
+    destination_type      = "local"
+    source_type           = "image"
+    uuid                  = data.openstack_images_image_v2.image_node.id
+  }
+
+  block_device {
+    boot_index            = -1
```

Comment:
Nova docs suggest to leave this unspecified for local devices.

Reply:
Does not work without this index:

```
│ Error: Error creating OpenStack server: Bad request with: [POST https://api.zone-1.pebblestones.cloud:8774/v2.1/servers], error message: {"badRequest": {"code": 400, "message": "Block Device Mapping is Invalid: Boot sequence for the instance and image/block device mapping combination is not valid."}}
```